src/pages/css/background-opacity/index.md

```diff
@@ -3,13 +3,12 @@ title: Background Opacity
 ---
 ## Background Opacity
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/css/background-opacity/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
-
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
-
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
-
-
+`opacity` is a css property that specifies the opacity or transparency factor of an element.
+```
+img {
+    opacity: 0.5;
+}
+```
+The above code snippet applies 50% opacity to an img element. The value of opacity property can range between 0 to 1. Higher the value of `opactiy`, lower is the transparency of the element.
```

Comment:
Opacity is often used to draw the user’s attention to elements and focus on important parts of a page.